os/booting-on-digitalocean.md

```diff
@@ -7,6 +7,8 @@ schedules per channel. You can [disable this feature][reboot-docs], although we
 don't recommend it. Read the [release notes][release-notes] for specific
 features and bug fixes.
 
+**NOTE: Deploying CoreOS on DigitalOcean is by standard not secure,** because DigitalOcean does not allow private networking. That means that even though you use private networking, you can still be attacked from others inside DigitalOceans private network.
```

Comment:
Sounds good. Can you modify the PR accordingly.

One note: I added the bold just to show what was new, I don't think we need to bold the whole thing :)